native-file-windows.txt

```diff
@@ -11,4 +11,5 @@ endian = 'little'
 [properties]
 c_args = ['-D_WIN32_WINNT=0x0601', '-march=x86-64']
 cpp_args = ['-D_WIN32_WINNT=0x0601', '-march=x86-64']
+nuget-download = true
```

Comment:
```suggestion
nuget-download = false
```
At this point we really don't need to enable this, and we could merge without drawbacks.